clients/drcachesim/tools/basic_counts.h

```diff
@@ -182,6 +182,12 @@ class basic_counts_t : public analysis_tool_t {
         int_least64_t encodings = 0;
         std::unordered_set<uint64_t> unique_pc_addrs;
 
+        // Metadata for the counts. These are not used for the equality, increment,
+        // or decrement operation, and must be set explicitly.
+
+        // Count of shards that were combined to produce the above counts.
+        int_least64_t shard_count = 1;
```

Comment:
This should be in the operator+=, operator-=, operator== too right?

Reply:
I documented above that this (and any future) metadata vars would not be included in those ops. It's not as simple as counts. E.g. we += counters_t not only across shards but across windows too.